Documentation/compatibility.md

```diff
@@ -35,3 +35,4 @@ The versions of Prometheus compatible to be run with the Prometheus Operator are
 * v2.2.1
 * v2.3.1
 * v2.3.2
+* v2.4.0
```

Comment:
Probably want to add 2.4.1 and 2.4.2 here as well

Reply:
I'd stick just with 2.4.2 due to `[BUGFIX] Handle WAL corruptions properly prometheus/tsdb#389`

Reply:
2.4.2 is the new default Prometheus version, this is just the compatibility matrix of supported versions by the operator.